luna/gateware/platform/numato_opsis.py

```diff
@@ -0,0 +1,208 @@
+#
+# This file is part of LUNA.
+#
+# Copyright (c) 2020 Marian Sauer
+# SPDX-License-Identifier: BSD-3-Clause
+
+""" Numato Opsis platform definitions.
+
+This is a non-core platform. To use it, you'll need to set your LUNA_PLATFORM variable:
+
+    > export LUNA_PLATFORM="luna.gateware.platform.numato_opsis:NumatoOpsisPlatform
+"""
+
+from nmigen import *
+from nmigen.build import *
+from nmigen.vendor.xilinx_spartan_3_6 import XilinxSpartan6Platform
+
+from nmigen_boards.resources import *
+from .core import LUNAPlatform
+
+__all__ = ["NumatoOpsisPlatform"]
+
+class NumatoOpsisClockDomainGenerator(Elaboratable):
+    """ NumatoOpsis clock domain generator.
+        Generate 60 MHz for ULPI phy in clock input operation.
+
+    """
+
+    def __init__(self, *, clock_frequencies=None, clock_signal_name=None):
+        pass
+
+    def elaborate(self, platform):
+        m = Module()
+
+        m.domains.sync = ClockDomain()
```

Comment:
Would you be able to align the `=`s and `.eq`s like in the other platform examples? Consistent style is nice. :)